### ISSUE TITLE
[Feature request] Build pip-installable wheels and distribute via pypi.

### ISSUE THREAD

**Issue by thatfloflo** [lart-bangor/research-client#38]
**Is your feature request related to a problem? Please describe.**
Currently, source installations are somewhat involved beyond what is strictly necessary, as people have to deal with the download and dependencies, when these could be taken care of by one of our dependencies itself, namely Python/pip.

**Describe the solution you'd like**
Integrate a build process to build wheels and release the `research_client` package via [PyPI](https://pypi.org/).

**Outlook**
Once this implemented, people on any platform that has Python and pip can in principle get the app running in just three steps, e.g.
```
sudo apt install python3-pip python3-tk
pip install research_client
python -m research_client
```
Where the last line will already run the app!

**Comment by thatfloflo** [comment]
48f23a1 implements the build mechanism and fixes that were needed for this to work. Overall the produced wheel and sdist look fine to me, but for some reason when installing it from TestPyPI it will not resolve the dependencies -- that could be a TestPyPI thing, but needs further looking into before taking this as closed.

The TestPyPI release is available here (use the latest to try, probably 0.3.6.postX: https://test.pypi.org/project/research-client

**Comment by thatfloflo** [comment]
Tested and works on proper PyPI -- presumably `pip` was trying to resolve the dependencies from TestPyPI where they don't exist.
